schemas/identity/identity-graph-flattened-export.schema.json

```diff
@@ -0,0 +1,108 @@
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Identity Graph Flattened Export Schema.",
+  "definitions": {
+    "identity-graph-flattened-export": {
+      "properties": {
+        "xdm:identityId": {
+          "title": "Person ID",
+          "type": "string",
+          "description": "Person ID Like Email or ECID or phone."
+        },
+        "xdm:namespaceId": {
+          "title": "Namespace Id",
+          "type": "integer",
+          "description": "Id for the Identity Type."
+        },
+        "xdm:namespaceCode": {
+          "title": "Namespace Code",
+          "type": "string",
+          "description": "Identity Type in human readable form."
+        },
+        "xdm:algo": {
+          "title": "Algorithm",
+          "type": "string",
+          "description": "Algorithm used by Identity Service to stitch like SHARED_DEVICE_DETECTION_SESSIONIZATION, DETERMINISTIC, FREQ_RECENCY etc.",
+          "enum": [
+            "SHARED_DEVICE_DETECTION_SESSIONIZATION",
+            "FREQ_RECENCY_BASED_SHARED_DEVICES",
+            "DETERMINISTIC"
+          ],
+          "meta:enum": {
+            "SHARED_DEVICE_DETECTION_SESSIONIZATION": "Shared Device Detection Sessionization",
+            "FREQ_RECENCY_BASED_SHARED_DEVICES": "Frequency Recency Based Shared Devices",
+            "DETERMINISTIC": "Deterministic"
+          }
+        },
+        "xdm:membersByTimeRange": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "xdm:startTs": {
+                "title": "Start Timestamp",
+                "type": "integer",
+                "minimum": -9223372036854776000,
+                "maximum": 9223372036854776000,
+                "description": "Start Timestamp."
+              },
+              "xdm:endTs": {
+                "title": "End Timestamp",
+                "type": "integer",
+                "minimum": -9223372036854776000,
+                "maximum": 9223372036854776000,
+                "description": "End Timestamp."
+              },
+              "xdm:graphId": {
+                "title": "Person Graph ID",
+                "type": "string",
+                "description": "Person level Graph Identifier."
+              },
+              "xdm:identityMap": {
+                "type": "object",
+                "meta:xdmType": "map",
+                "additionalProperties": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "https://ns.adobe.com/xdm/context/identityitem"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "xdm:lastUpdatedTime": {
+          "title": "Last Updated Time",
+          "type":"string",
+          "format":"date-time",
+          "description": "Last updated time when this identity was updated."
```

Comment:
Is this column something we could leverage when computing the deltas in the future? For instance, instead of joining both the old & new snapshots, maybe we could load only the newer snapshot and look at the `lastUpdatedTIme`?

Reply:
Yes, for first phase using full daily export, this may not be useful. But when we move to incremental exports in second phase, we will use this field to help Siphon handle Out-of-order processing of messages.
Platform-SDK will provide interfaces to get deltas based on time-stamps.

Reply:
Cool, that makes sense :)